services/backend/src/db/seedData.js

```diff
@@ -24,6 +24,13 @@ const USERS = [
         password: "$2a$13$ou.iRf4nlSfcXp0lRewliuxbp8wD9M8/CIy6pomcbL.EB5CrZMZze",  // Password = 'test'
         createdAt: new Date(),
         updatedAt: new Date()
+    },
+    {
+        id: "07a4d430-7f9e-466a-b671-8076468d84e6",
```

Comment:
I'm just going to remove this for now in this case since this can probably be done in bulk when implementing the sign up process to connect users to profiles.

I only needed it at the time to test the `restrictToUser` hook so it isn't currently doing anything if it was to be on prod.